lib/tasks/migrations/migrations.rake

```diff
@@ -121,6 +121,91 @@ namespace :migrations do
     exit has_encountred_issue
   end
 
+  task :room_settings, [:start, :stop] => :environment do |_task, args|
+    start, stop = range(args)
+    has_encountred_issue = 0
+
+    filtered_roles_ids = Role.unscoped
+                             .select(:id, :name)
+                             .where(name: COMMON[:filtered_user_roles])
+                             .pluck(:id)
+
+    Room.unscoped
+        .select(:uid, :room_settings)
+        .where.not(users: { role_id: filtered_roles_ids, deleted: true }, deleted: true)
+        .find_each(start: start, finish: stop, batch_size: COMMON[:batch_size]) do |r|
+
+      parsed_room_settings = JSON.parse(r.room_settings)
+
+      # Returns nil if the Room Setting value is the same as the corresponding default value in V3
+      room_settings = {
+        record: parsed_room_settings["recording"] == false ? nil : "true",
+        muteOnStart: parsed_room_settings["muteOnStart"] == false ? nil : "true",
+        glAnyoneCanStart: parsed_room_settings["anyoneCanStart"] == false ? nil : "true",
+        glAnyoneJoinAsModerator: parsed_room_settings["joinModerator"] == false ? nil : "true",
+        guestPolicy: parsed_room_settings["requireModeratorApproval"] == false ? nil : "ASK_MODERATOR",
+      }
+
+      params = { room: { friendly_id: r.uid, room_settings: room_settings.compact } }
```

Comment:
Made it so only RoomSettings values that differs from the default values of V3 are sent

Reply:
`room_settings.compact` on line 149 will remove the keys with nil value

Reply:
v2 seems to support only "ASK_MODERATOR" (and not "ALWAYS_ACCEPT") according to line 78 in bbb_server.rb